region/tourian/main/Metroid Room 3.json

```diff
@@ -945,20 +996,51 @@
       },
       "requires": [
         "canMetroidAvoid",
+        "canTrickyDodgeEnemies",
         "h_artificialMorphSpringBall",
-        "h_artificialMorphBombHorizontally",
-        {"ammo": {"type": "PowerBomb", "count": 3}}
+        {"or": [
+          {"tech": "canSpringBallBombJump"},
```

Comment:
Should this include an extra bomb for the springballbombjump?
Also springfling isn't listed.